src/main/java/com/dougnoel/sentinel/exceptions/NumberFormatException.java

```diff
@@ -0,0 +1,44 @@
+package com.dougnoel.sentinel.exceptions;
+
+/**
+ *  Wrapping java.lang.NumberFormatException so that it is a RuntimeException.
+ */
+public class NumberFormatException extends RuntimeException {
```

Comment:
Is this being used anywhere besides the test file?

Reply:
Yes, actually. It's being used in the Verification steps when a number format exception occurs if the inputs cannot be parsed as doubles or the value returned from the configuration key is null.